src/explore-education-statistics-admin/src/pages/release/data/__tests__/ReleaseApiDataSetChangelogPage.test.tsx

```diff
@@ -194,6 +194,36 @@ describe('ReleaseApiDataSetChangelogPage', () => {
     });
   });
 
+  test('renders warning if unable to fetch data set', async () => {
+    apiDataSetService.getDataSet.mockRejectedValue(
+      new Error('Unable to fetch changes'),
+    );
+    apiDataSetVersionService.getChanges.mockResolvedValue(testChanges);
+
+    renderPage('draft-version-id');
+
+    expect(screen.queryByText('Data set title')).not.toBeInTheDocument();
+
+    expect(
+      await screen.findByText('Could not load changelog'),
```

Comment:
As per other comments - this would be 'Could not load API data set'

Reply:
Done ta